code/_helpers/game.dm

```diff
@@ -20,7 +20,7 @@
 /proc/living_observers_present(var/list/zlevels)
 	if(LAZYLEN(zlevels))
 		for(var/mob/M in GLOB.player_list) //if a tree ticks on the empty zlevel does it really tick
-			if(M.stat != DEAD) //(no it doesn't)
+			if(istype(M, /mob/observer) || (M.stat != DEAD)) //(no it doesn't)
```

Comment:
Best practice to put the least-expensive checks first, that way short circuiting is more useful.
```suggestion
			if(M.stat != DEAD || istype(M, /mob/observer)) //(no it doesn't)
```

Reply:
Done.